jakarta.batch.tck.util/src/main/java/ee/jakarta/tck/batch/util/PropertyKeys.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package ee.jakarta.tck.batch.util;
+
+public interface PropertyKeys {
+
+    String PROPERTIES_FILE_NAME = "vehicle.properties";
+    /* A comma-separated list of vehicles to run the tests with. Possible values: ejb, web, standalone */
+    String ENABLED_VEHICLE = "vehicle";
```

Comment:
It'd probably be worth prepending the prefix: **arquillian.extensions.jakarta.batch**
from
```java
public class ArquillianExtension implements LoadableExtension {

    public static final String PROPERTY_PREFIX = "arquillian.extensions.jakarta.batch.";
```

Though it's pretty unlikely there'd be a collision, we are starting a server which might have a whole set of system props associated with it.

Reply:
But I'm too lazy to do this right now so not going to suggest you have to.  Just commenting.